docs/demos/kubeadm-bootstrap/kubeadm-bootstrap.sh

```diff
@@ -24,29 +25,20 @@ function log() {
     echo "# $(date --rfc-3339=ns) :: ${*}" >&2
 }
 
-function kubeadm_init_phase_certs() {
-    while read cert; do
-        log "Generating CSR ${cert}"
-        local args=""
-        if [[ "${cert}" == "apiserver" ]]; then
-            args="--apiserver-cert-extra-sans=kind-control-plane,127.0.0.1"
-        fi
-        ${KUBEADM} init phase certs "${cert}" ${args} --csr-only --csr-dir "${CERTIFICATES_DIR}" --cert-dir "${CERTIFICATES_DIR}" >/dev/null
-        echo "${cert} ${CERTIFICATES_DIR}/${cert/#etcd-/etcd\/}.csr"
-    done
-}
-
 function vcert_enroll() {
     while read nickname csr; do
         cert="${csr/%.csr/.crt}"
-        policy="$(echo "${nickname}" | grep -o '\(client\|server\|peer\)$')"
+        policy="$(echo "${nickname}" | grep -o '\(client\|server\|peer\)$' || true)"
         # Special case for etcd-server which needs both server and client usage
         # See https://clusterise.com/articles/kbp-2-certificates/ and
         # https://kubernetes.io/docs/setup/best-practices/certificates/#all-certificates
         if [[ "${nickname}" == "etcd-server" ]]; then
             policy=peer
         fi
-        log "Enrolling CSR ${csr} with nickname ${nickname} to ${cert}"
+        if echo "${nickname}" | grep '\.conf$' > /dev/null; then
```

Comment:
use `if [[ "${nickname}" == *\.conf ]]`?
Not too sure about bash syntax actually, have a look at https://www.diskinternals.com/linux-reader/bash-string-ends-with/